sdv/vdb/client.py

```diff
@@ -41,16 +40,20 @@ class VehicleDataBrokerClient:
     def __new__(cls, port: Optional[int] = None):
```

Comment:
Why is this `__new__` and not `__init__`?

Reply:
This is used for the Singleton implementation of the VDBClient